### PR TITLE
debitcard: add debit card option to payment method

### PR DESCRIPTION
**Pull request by vitorabner**
In our library there is no option to debit_card payment method. We need to add this option. For the ESRB customer or use it in a totem application.

### CODE REVIEW

File: src/main/java/me/pagar/model/Transaction.java [hunk 1]
```diff
@@ -206,13 +298,20 @@ public void setCardCvv(String cardCvv) {
     private AcquirerName acquirerName;
 
     /**
-     * Método de pagamento possíveis: <code>credit_card</code> e <code>boleto</code>
+     * Métodos de pagamento possíveis: <code>credit_card</code> e <code>boleto</code>
```
Comment (grvcoelho) [inline src/main/java/me/pagar/model/Transaction.java]: @vitorabner: faltou colocar `debit_card` :smile:

Comment (vitorabner) [inline src/main/java/me/pagar/model/Transaction.java]: boa!

File: src/main/java/me/pagar/CaptureMethod.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+public enum CaptureMethod {
+    
+    ecommerce("ecommerce"),
+    emv("emv"),
+    magstripe("magstripe");
+    
+    private String method;
+
+    private CaptureMethod(String method) {
+            this.method=method;
```
Comment (thalesmello) [inline src/main/java/me/pagar/CaptureMethod.java]: Space between equal sign missing

Comment (vitorabner) [inline src/main/java/me/pagar/CaptureMethod.java]: Done

Comment (thalesmello) [inline src/main/java/me/pagar/CaptureMethod.java]: :+1:

**Pull request merged**